server/controllers/notificationController.js

```diff
@@ -1,17 +1,106 @@
+import dotenv from 'dotenv';
+import models from '../models';
+
+const { Notification } = models;
+
+dotenv.config();
 /**
- * This class contains method to handle notifications route
- * required to handle notifications
+ * This class contains all methods
+ * required to handle
+ * accommodations' request.
  */
-class NotificationController {
+class Notifications {
+  /**
+   * This method handle the accommodation request.
+   * @param {object} req The notification's request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the accomodation.
+   */
+  static async viewNotifications(req, res) {
+    const foundNotice = await Notification.findAll({
+      where: {
+        receiverId: req.user.id
+      }
+    });
+    if (foundNotice.length === 0) {
+      return res.status(404).json({
+        status: 404,
+        error: 'You dont have any notifications'
+      });
+    }
+    return res.status(200).json({
+      status: 200,
+      data: foundNotice
+    });
+  }
+
   /**
-     * This method handle the html file.
-     * @param {object} req The user's request.
-     * @param {object} res The user's response.
-     * @returns {object} The status and message.
-     * */
-  static async userNotification(req, res) {
-    return res.sendFile(`/notification.html`, { root: 'UI/html' });
+   * This method handle the accommodation request.
+   * @param {object} req The notification's request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the accomodation.
+   */
+  static async markOne(req, res) {
+    const foundOneNotice = await Notification.findOne({
+      where: {
+        receiverId: req.user.id,
+        id: req.params.id
+      }
+    });
+    if (!foundOneNotice) {
+      return res.status(404).json({
+        status: 404,
+        error: 'Notification not found'
+      });
+    }
+    if (req.body.viewed === 'read') {
+      await Notification.update({ viewed: 'read' }, { where: { id: req.params.id } });
+      return res.status(200).json({
+        status: 200,
+        message: 'Notification marked as read'
+      });
+    }
+    if (req.body.viewed === 'unread') {
```

Comment:
Hi @manzi-guev nice try. However, your if conditions are messy. 
1. why two if statements? we are dealing with a boolean. 
2. for line 56: it is very confusing. the variable viewed is already in table as a field for Notification. so I recommend you to use an other, like readFlag